package.json

```diff
@@ -97,7 +97,7 @@
     "jest-environment-jsdom-fifteen": "^1.0.2",
     "lint-staged": "^10.5.4",
     "lz-string": "^1.4.4",
-    "npm": "^6.14.7",
+    "npm": "^8.12.1",
```

Comment:
Please try to keep "NPM" version at least on same major version that is "6.x", And "6.14.7" does not contain vulnerabilities so try with that version once.